pkg/logs/decoder/decoder.go

```diff
@@ -67,21 +66,29 @@ func NewMessage(content []byte, status string, rawDataLen int, timestamp string)
 	}
 }
 
-// Decoder splits raw data into lines and passes them to a lineParser that passes them to
-// a lineHandler that emits outputs
-// Input->[decoder]->[parser]->[handler]->Message
+// Decoder wraps a collection of internal actors, joined by channels, representing the
```

Comment:
Do you think it would make sense to duplicate Decoder description from the README here?

Reply:
I'm not sure which README you're referring to?
```
tereshkova ~/go/src/github.com/DataDog/datadog-agent [ac-1537-b] $ git ls-files | grep README.md | xargs grep Decoder
pkg/logs/README.md:│ │      Decoder      │  ─┘
pkg/logs/README.md:Decoders and their components are defined in `pkg/logs/decoder`.
```

Reply:
The one you change in this PR. 😄 You added a nice description of a decoder, which I think would work well as the first doc line here as well:

```
A decoder translates a sequence of byte buffers (such as from a file or a network socket) into log messages.
```

Reply:
Ah, got it -- I was looking for something bigger :)